src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ CHANGELOG
  * Require explicit argument when calling `Definition::setInitialPlaces()`
  * `GuardEvent::getContext()` method has been removed. Method was not supposed to be called within guard event listeners as it always returned an empty array anyway.
  * Remove `GuardEvent::getContext()` method without replacement
+ * Add partial support for marking with more than one token per place
```

Comment:
the word "partial" means nothing to the reader i think (it doesn't to me at least)

Reply:
I don't know how to said that shortly but

There was an issue with the previous version of the workflow. see #53179. This PR fix this case, but open the door to real multi token per place. 

In another PR (much more bigger) I planned to add a real support. Symfony will expose a `weight` attribute on each transition, and this value will be used during the `apply` and `can` method.

Reply:
Well, the changelog should describe what is supported. Saying `partial support` does not explain what gets added (maybe that's supporting the whole feature set but only during full moon, which would also make it a partial support :trollface:)

Reply:
Maybe this is a bug fix instead. WDYT ?